src/main/java/frc/robot/auto_manager/AutoManager.java

```diff
@@ -169,18 +168,8 @@ private boolean robotInBox() {
     return getScoringBox().contains(localization.getUsedPose().getTranslation());
   }
 
-  private Command intakeNoteAtPose(Supplier<Optional<Translation2d>> pose) {
-    // TODO: THIS IS BAD AND BROKEN. STOP CHECKING IF A NOTE IS THERE WHEN THE COMMAND IS
-    // CONSTRUCTED. HAVE TO CHECK WHEN COMMAND IS SCHEDULED.
-    Optional<Translation2d> maybeSearchArea = pose.get();
-    if (maybeSearchArea.isPresent()) {
-      DogLog.log("Debug/IntakeSearchPose", maybeSearchArea.get());
-      return noteTrackingManager.intakeNoteAtPose(() -> pose.get(), 1.5);
-    } else {
-
-      DogLog.log("Debug/IntakeSearchPose", "null :(");
-    }
-    return Commands.none();
+  private Command intakeNoteAtPose(Supplier<Optional<Translation2d>> searchPoseSupplier) {
+    return noteTrackingManager.intakeNoteAtPose(searchPoseSupplier, 1.5);
```

Comment:
removes the debug logging & implements the behavior described in the TODO, creating a command that intakes a note at a search pose (which is provided via a Supplier)